Tests/BackendIntegrationTests/OfflineStoreKitIntegrationTests.swift

```diff
@@ -243,19 +243,16 @@ class OfflineStoreKit1IntegrationTests: BaseOfflineStoreKitIntegrationTests {
 
     @available(iOS 15.0, tvOS 15.0, watchOS 8.0, macOS 12.0, *)
     func testCallToGetCustomerInfoWithPendingTransactionsPostsReceiptOnlyOnce() async throws {
-        // forceRenewalOfSubscription doesn't work well, so we use this instead
-        setShortestTestSessionTimeRate(self.testSession)
```

Comment:
I reverted this change before 5.0 but was brought back when we merged 5.0. This was causing tests to fail in my machine and CI